fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -81,25 +82,36 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
 
 /**
  * EditLog maintains a log of the memory modifications.
  * Current we support only file editLog.
  */
 public class EditLog {
     public static final Logger LOG = LogManager.getLogger(EditLog.class);
+    private static final int OUTPUT_BUFFER_INIT_SIZE = 128;
 
-    private final EditLogOutputStream editStream = null;
-
-    private long txId = 0;
+    private final Journal journal;
 
-    private long numTransactions;
-    private long totalTimeTransactions;
+    private BlockingQueue<JournalQueueEntity> logQueue;
 
-    private final Journal journal;
+    /**
+     * for ut only
```

Comment:
How about annotate with `Visible Test`?

Reply:
Good advice. I didn't know this annotation.